tests/platform/test_platform.py

```diff
@@ -693,7 +693,7 @@ def test_execute_qprogram_with_qblox(self, platform: Platform):
 
         # assert run executed all three times (6 because there are 2 buses)
```

Comment:
comment says 6 because there are two buses, now with the channel id it went to 9, why? can we specify it in the test?

Reply:
I am checking right now why, I wanted to see if there was anything else in code coverage to handle

Reply:
Answered above